qt/widgets/instrumentview/src/UnwrappedSurface.cpp

```diff
@@ -443,7 +443,18 @@ void UnwrappedSurface::setPeaksWorkspace(const std::shared_ptr<Mantid::API::IPea
   po->setShowRowsFlag(m_showPeakRows);
   po->setShowLabelsFlag(m_showPeakLabels);
   po->setShowRelativeIntensityFlag(m_showPeakRelativeIntensity);
-  m_peakShapes.append(po);
+
+  bool containsPeak = false;
+  for (auto pkShape : m_peakShapes) {
```

Comment:
Would it be possible to use the `any_of` standard library algorithm for this? https://cplusplus.com/reference/algorithm/any_of/